server/api/utils/helpers.py

```diff
@@ -165,3 +165,96 @@ def wrapper(*args, **kwargs):
         return wrapper
 
     return decorator
+
+
+def set_scheduled_object_labels(
```

Comment:
I think this methods can be `staticmethods` of the Scheduler class it would make more sense to me

Reply:
@alonmr then we will have circular imports ad this method is used in `helpers` below.